components/MainMap/MapFeatures.vue

```diff
@@ -389,13 +390,8 @@ export default defineNuxtComponent({
 
       const currentZoom = this.map.getZoom()
 
-      if (
-        !isOneInView
-        && currentZoom >= MAP_ZOOM.zoom.default
-        && this.features.length > 0
-      ) {
+      if (!isOneInView && currentZoom >= MAP_ZOOM.zoom.default && this.features.length > 0)
```

Comment:
All formating changes are linked to the use of an other eslint version ?

Reply:
No i did it myself, but you are right I shouldn't. Once ESLint is configured properly it will handle it :)